tests/src/test/java/org/eclipse/hono/tests/registry/TenantHttpIT.java

```diff
@@ -214,6 +240,12 @@ public void testUpdateTenantFailsForNonExistingTenant(final VertxTestContext con
     public void testRemoveTenantSucceeds(final VertxTestContext context) {
 
         final Tenant tenantPayload = buildTenantPayload();
+        final var trustedCa = new TrustedCertificateAuthority()
```

Comment:
what is this good for?

Reply:
nothing really, just added an additional config for the tenant payload